examples/storagetransfer-ts/index.ts

```diff
@@ -0,0 +1,112 @@
+import * as gcp from "@pulumi/gcp";
+import * as gnative from "@pulumi/google-native";
+import * as storage from "@pulumi/google-native/storage/v1";
+import * as pulumi from "@pulumi/pulumi";
+import * as random from "@pulumi/random";
+import * as fs from "fs";
+import * as mime from "mime";
+
+const randomString = new random.RandomString("name", {
+    upper: false,
+    number: false,
+    special: false,
+    length: 5,
+});
+
+const config = new pulumi.Config("google-native");
+const project = config.require("project");
+const bucketName = pulumi.interpolate`pulumi-transfer-test-dst-${randomString.result}`;
+const transferJobName = "transfer-test";
+
+// Create a Google Cloud resource (Storage Bucket)
+const bucket = new storage.Bucket("my-bucket", {
+    name: bucketName,
+    project: project,
+});
+
+const storageTransferServiceAccount = gcp.storage.getTransferProjectServieAccount({
+    project: project,
+});
+
+const iam = new gcp.storage.BucketIAMMember("iam-dest", {
```

Comment:
I was trying to create this with the native provider using the following code but :
```
const iam = new storage.BucketIamPolicy("iam-dest", {
    bucket: bucket.name,
    bindings: [{
        members: [pulumi.interpolate `serviceAccount:${storageTransferServiceAccount.then(sa => sa.email)}`],
        role: "roles/storage.admin",
    }]
});
```

but this overwrites the members and makes the bucket unusable. Am I doing something wrong @mikhailshilkov? Otherwise will open an issue to track this.

Reply:
Cut https://github.com/pulumi/pulumi-google-native/issues/118